### ISSUE TITLE
NEO TOKYO

### ISSUE THREAD

**Issue by oggreo** [decentraland/districts#83]
![image](https://user-images.githubusercontent.com/5387810/33883083-bcfa34dc-df7d-11e7-9606-7aadda2a9824.png)

** Goal **
TO BECOME NINJA!!!

**Concept**
NEO TOKYO
This is a world where you can become Ninja!
The idea is to bring back ninja into the future cybernetic world. So I imagine the city to be a hybrid of traditional Japanese cultures and sky scrapers filled with neon lights. Almost like the Neo Tokyo of Akira mixed with city scape of Konoha in Naruto and such. I am not going to be too strict on the theme so anyone interested in either one of them can participate freely in the world building. This urban design of traditional and future Tokyo mixed will make this an exciting world to roam.

DOJO (THE NINJA TEST)
Here you can train and take test to become a ninja. The test will give you trials to see if you have what it got to be a ninja. In Japan there really exists a ninja test. This will be a virtual version of that. It will be really cool if I can collaborate with a school of ninja that exists in RL. ( I hear not many people wants to be ninja nowadays… It’ll be a nice way to preserve tradition). In Dojo if you pass the test you are given “Makimono”(scrolls) which is a certificate for ninja. I can use the block chain to prove the authenticity of the certificate. There will be certain ninja outfits and weapons like katanas where only the holder of “Makimono" can purchase.

SASUKE (NINJA ARENA)
Ninja fights? Ninja missions? Ninja tags? Shuriken shooter? There are so many possibilities in this world. Open for discussions!!

**Category**
Public

**LAND Needed**
 50. More the merrier.

**Return**
If you are here from the start you can become the Hokage of NEO TOKYO. 

**TEAM**
Reo
Tatsuya
Taiki
Neotokyodecentra@gmail.com

Members and participants needed!

![image](https://user-images.githubusercontent.com/5387810/33883109-e00ec672-df7d-11e7-9ff6-e0800642d183.png)

**Comment by ttyszk** [comment]
This is super cool!

**Comment by hrsma2i** [comment]
Wow, awsome

**Comment by pavel-corsaghin** [comment]
Hi, i have only one parcel at (22, -100). Can i join this project?

**Comment by oggreo** [comment]
@pavel-corsaghin 
Yeah of course! but committing LAND to the district is not allowed at this moment since the terraform event is over. I think it will be possible when they open second market for LAND trading. You are most welcome to join this project though we are just starting to plan out how we can make NEO Tokyo happen. I'm thinking about opening a chat on rocketchat. Ill let you know when I do.
